### PR TITLE
[keepalived-vip]: Update keepalived and godeps

### PR DESCRIPTION
**Pull request by aledbf**

### CODE REVIEW

**Comment by aledbf** [comment]
ping @bprashanth

File: keepalived-vip/Makefile [hunk 1]
```diff
@@ -6,7 +6,7 @@ PREFIX = gcr.io/google_containers/kube-keepalived-vip
 BUILD_IMAGE = build-keepalived
 
 controller: clean
-	CGO_ENABLED=0 GOOS=linux godep go build -a -ldflags '-w' -o kube-keepalived-vip
+	CGO_ENABLED=0 GOOS=linux go build -a -ldflags '-w' -o kube-keepalived-vip
```
Comment (bprashanth) [inline keepalived-vip/Makefile]: this will use gopath right? or am i missing something

Comment (aledbf) [inline keepalived-vip/Makefile]: yes

Comment (bprashanth) [inline keepalived-vip/Makefile]: why do we want that? hermetic builds are good, no?

Comment (aledbf) [inline keepalived-vip/Makefile]: I'm sorry, in go 1.6 uses the vendor dir (if exists)

Comment (eparis) [inline keepalived-vip/Makefile]: Just FYI: godep also used GOPATH. It just used Godep/_workspace first. Just like 1.6+vendor will use vendor/ first.

Comment (bprashanth) [inline keepalived-vip/Makefile]: Ah, yes

**Comment by bprashanth** [comment]
LGTM

**Pull request merged**